src/components/ProductList/index.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react'
+import ProductCard from '../ProductCard'
 
 const ProductList = () => {
+  const articles = new Array(20).fill(<ProductCard />)
```

Comment:
I just found that the solutions based on fill (B,C) are fast on Chrome and Safari but surprisingly slowest on Firefox for big arrays. Please check this [https://stackoverflow.com/questions/1295584/most-efficient-way-to-create-a-zero-filled-javascript-array](url)